Package.swift

```diff
@@ -295,6 +295,7 @@ let package = Package(
                 .target(name: "WireGuardC"),
                 .product(name: "WireGuard", package: "wireguard-apple"),
                 "Common",
+                "Subscription"
```

Comment:
I needed this to get access to the specific error being thrown by the token store

Reply:
It's fine for here, but one thing worth noting is we should push to remove the token store entirely from BSK, at some point.  Mostly as it's only necessary in macOS sysex.

Not sure you agree, but if you do let me know and I'll create a quick win task for this.